Polyhedron/demo/Polyhedron/Plugins/Convex_decomposition/Nef_plugin.cpp

```diff
@@ -56,6 +56,7 @@ void init(QMainWindow* mw,
       action->setProperty("subMenuName",
                           "Boolean Operations");
     _actions<< actionConvexDecomposition;
+    connect(actionConvexDecomposition, SIGNAL(triggered()), this, SLOT(on_actionConvexDecomposition_triggered()));
```

Comment:
Given the names of the action and of the slot, was not that connection made with the auto-connect feature?